### PR TITLE
Feat/caseread

### PR DESCRIPTION
**Pull request by seanpearsonuk**
- a Fluent case reader to facilitate the local parametric study [here ](https://github.com/pyansys/pyfluent-parametric/pull/47), and to allow launch fluent based on case file args (to follow) 
- lispy.py. This provides a Scheme parser, **copied from the web**. It was necessary to make a few adjustments. Null / empty list triggered an exception, while (1 . 2) was interpreted as [1, '.', 2], for which I generate [1, 2].

### CODE REVIEW

File: src/ansys/fluent/core/filereader/casereader.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    Methods
+    -------
+    input_parameters
+        Get a list of input parameter objects
+    output_parameters
+        Get a list of output parameter objects
+    """
+
+    def __init__(self, case_filepath: str):
+        file = h5py.File(case_filepath)
```
Comment (mkundu1) [inline src/ansys/fluent/core/filereader/casereader.py:72]: Does this work with legacy .cas and .cas.gz files?

Comment (seanpearsonuk) [reply src/ansys/fluent/core/filereader/casereader.py:72]: @mkundu1 thanks for reminding me. I intended to put a check for that.

Comment (seanpearsonuk) [reply src/ansys/fluent/core/filereader/casereader.py]: ...and we can also work on legacy support. Adding issue.

File: src/ansys/fluent/core/launcher/launcher.py [hunk 2]
```diff
@@ -184,6 +184,7 @@ def launch_fluent(
     cleanup_on_exit: bool = True,
     start_transcript: bool = True,
     show_gui: bool = None,
+    case_filepath: str = None,
```
Comment (seanpearsonuk) [inline src/ansys/fluent/core/launcher/launcher.py:187]: @mkundu1 @ajain-work @hpohekar @tejalprabhu01 

@prmukherj has merged his work into this branch: launching by case file path.

**Review by mkundu1 — APPROVED** [review]

**Review by hpohekar — APPROVED** [review]

**Review by tejalprabhu01 — APPROVED** [review]

**Review by prmukherj — APPROVED** [review]
.h5 reader is running fine. Case file can also be loaded to fluent with an extra argument added in the launcher.

**Pull request merged**